### PR TITLE
Added values to chose init container, additional label for Ingress and deleteDefaultPlugins option.

### PR DESCRIPTION
**Pull request by andreyshamray**
#### What this PR does / why we need it:
I have added several values which will improve the flexibility, security and upgrade plugins procedure of this chart.
1. ingress.labels - will give you an opportunity to route traffic to particular Ingress controller. 
2. plugins.initContainerImage - it is needed for security reason, for example, access to docker hub is not permitted.
3. plugins.deleteDefaultPlugins - this option will remove default plugins and deploy plugins from plugins.install list. It will give an opportunity to upgrade plugins.

### CODE REVIEW

**Comment by andreyshamray** [comment]
Hi, guys will be good if you can merge it, so I will no need to store this chart in local helm repo.
Thanks in advance.

File: stable/sonarqube/values.yaml [hunk 1]
```diff
@@ -34,6 +34,11 @@ ingress:
   # kubernetes.io/tls-acme: "true"
   # This property allows for reports up to a certain size to be uploaded to SonarQube
   # nginx.ingress.kubernetes.io/proxy-body-size: "8m"
+
+#Additional labels for Ingress manifest file
+  #labels:
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml]: Please add a space between the # and the comment

File: stable/sonarqube/values.yaml [hunk 2]
```diff
@@ -103,6 +108,9 @@ persistence:
 #    - "https://github.com/SonarSource/sonar-ldap/releases/download/2.2-RC3/sonar-ldap-plugin-2.2.0.601.jar"
 plugins:
   install: []
+
+  #initContainerImage: alpine:3.9
+  #deleteDefaultPlugins: true
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml]: Same here, please add a space after the #

File: stable/sonarqube/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.13.4
+version: 0.13.5
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please bump the minor version as this adds new features

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by andreyshamray** [comment]
I have applied all the recommendations.
Thanks!

**Comment by andreyshamray** [comment]
@rjkernick could you pls help to understand what is wrong again?)
Thanks a lot.

**Comment by rjkernick** [comment]
Please merge the latest master and update the chart.yaml. Please be sure to include "--signoff" when merging. Another pull request got merged ahead of yours

**Comment by andreyshamray** [comment]
> Please merge the latest master and update the chart.yaml. Please be sure to include "--signoff" when merging. Another pull request got merged ahead of yours

Done!)

**Comment by andreyshamray** [comment]
Thanks a lot!

**Pull request merged**